osp/core/session/wrapper_session.py

```diff
@@ -42,12 +42,38 @@ def check_consumes_buffers(func):
         and func.does_consume_buffers
 
 
+class DisableLazyReload():
+    """Context manager that disables lazy reload of expired objects."""
+    def __init__(self):
+        """Safe the previous setting."""
+        self.prev = None
```

Comment:
Should this be `self.pref`?

Reply:
Nice spot! I think this mistake doesn't break anything and probably we could even remove the __init__ altogether.,,
But I agree, if we have it it should use the same variable name as __enter__ and __exit__....

I'll change everything to prev as an abbreviation for previous

Reply:
Also, in the docstring it should be _save_ ;)

Reply:
Aaaaaaaaargh :)

Reply:
Fixed as well :)